### PR TITLE
Add dbcdata.sql

### PR DESCRIPTION
**Pull request by TheGrimsey**
The CreateDatabase script goes through and runs the template files to set up the databases.

Also added dbcdata.sql which just creates an empty database called "dbcdata.sql" (Stops the server from complaining about it being missing on start up)

### CODE REVIEW

File: resources/Database Files/Templates/dbcdata.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET NAMES utf8 */;
+/*!50503 SET NAMES utf8mb4 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+
+-- Dumping database structure for worldserver
```
Comment (NixAJ) [inline resources/Database Files/Templates/dbcdata.sql]: This should state dbcdata

Comment (TheGrimsey) [reply resources/Database Files/Templates/dbcdata.sql]: oops. Too fast on the copy paste work.

**Comment by TheGrimsey** [comment]
Corrected it.

**Comment by NixAJ** [comment]
Merged. Thank you :)

**Pull request merged**